python/sglang/srt/server_args.py

```diff
@@ -41,8 +41,9 @@ class ServerArgs:
     additional_ports: Optional[Union[List[int], int]] = None
 
     # Memory and scheduling
+    # A default value "None" means a computed default value will be used.
     mem_fraction_static: Optional[float] = None
-    max_prefill_tokens: Optional[int] = None
+    max_prefill_tokens: int = 16384
```

Comment:
@merrymercy I can change this back to None and add some code to automatically compute this value to avoid the OOM. Can you let me know what the best way to compute this value? much thanks!

Reply:
this looks good. We will use chunked prefill to solve this ultimately